src/ImageSharp/Formats/Gif/GifDecoderCore.cs

```diff
@@ -332,6 +333,7 @@ private void ReadFrame()
         /// </summary>
         /// <param name="imageDescriptor">The <see cref="GifImageDescriptor"/>.</param>
         /// <param name="indices">The pixel array to write to.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
There's no harm in doing this. We want to inline as many small private methods as we can really.